### PR TITLE
Feature/456918 retrieve producer regulator comments latest

### PR DESCRIPTION
**Pull request by prasadramireddyeviden**
Feature/456918 retrieve producer regulator comments latest

### CODE REVIEW

File: src/EPR.RegulatorService.Facade.Core/Helpers/TestData/ServiceRole.cs
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Facade.Core/Helpers/TestData/ServiceRole.cs:1]: needs to be in an enums folder and in a namespace

File: src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationDetails.cs
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationDetails.cs:1]: this is a response, not a request, can you relocate it?

File: src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationSubmissionSummaryDetails.cs
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationSubmissionSummaryDetails.cs:1]: a response...

File: src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationType.cs
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationType.cs:1]: move to enums folder

File: src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionStatus.cs
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionStatus.cs:1]: move to enums

File: src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionsOrganisationPaymentDetails.cs
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionsOrganisationPaymentDetails.cs:1]: not a request

File: src/EPR.RegulatorService.Facade.Core/Services/CommonData/ICommonDataService.cs
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Facade.Core/Services/CommonData/ICommonDataService.cs:1]: this needs to be renamed to GetOrganisationRegistrationSubmissionDetails, to partner the new method GetOrganisationRegistrationSubmissions

**Review by JulianYoungEviden — CHANGES_REQUESTED** [review]
some code movement required...

File: src/EPR.RegulatorService.Facade.Core/Helpers/TestData/ServiceRole.cs
Comment (prasadramireddyeviden) [reply src/EPR.RegulatorService.Facade.Core/Helpers/TestData/ServiceRole.cs:1]: it's done

File: src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationSubmissionSummaryDetails.cs
Comment (prasadramireddyeviden) [reply src/EPR.RegulatorService.Facade.Core/Models/Requests/RegistrationSubmissions/RegistrationSubmissionOrganisationSubmissionSummaryDetails.cs:1]: It's done

File: src/EPR.RegulatorService.Facade.Core/Enums/ServiceRole.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel;
+
```
[the remark this answers is not in the document]
Comment (prasadramireddyeviden) [reply src/EPR.RegulatorService.Facade.Core/Enums/ServiceRole.cs:2]: it's update. Thanks.

File: src/EPR.RegulatorService.Facade.Core/Services/CommonData/ICommonDataService.cs [hunk 2]
```diff
@@ -10,4 +11,6 @@ public interface ICommonDataService
     Task<HttpResponseMessage> GetPoMSubmissions(GetPomSubmissionsRequest pomSubmissionsRequest);
     
     Task<HttpResponseMessage> GetRegistrationSubmissions(GetRegistrationSubmissionsRequest registrationSubmissionsRequest);
+
+    Task<HttpResponseMessage> GetOrganisationRegistrationSubmissions(Guid submissionId);
```
Comment (shehzad-ismail8) [inline src/EPR.RegulatorService.Facade.Core/Services/CommonData/ICommonDataService.cs:15]: Can we rename this to GetOrganisationRegistrationSubmissionDetails?

**Review by shehzad-ismail8 — CHANGES_REQUESTED** [review]
Minor changes

**Review by shehzad-ismail8 — APPROVED** [review]

**Review by JulianYoungEviden — APPROVED** [review]

**Pull request merged**